src/command/cohost_archive.rs

```diff
@@ -107,14 +110,16 @@ pub async fn main(args: CohostArchive) -> eyre::Result<()> {
         .collect::<Vec<_>>();
 
```

Comment:
if the user runs this with both `--liked` and specific projects, what should we do if none of them are the project you’re logged in as? maybe `bail!()` with an error?

Reply:
part of me wants to say the flag would simply be a no-op in this case - given the help text says "your" posts specifically i'm not sure this warrants bailing, but i'm open to other ideas. maybe a warning?

Reply:
yea, warning works